packages/serverless-component/README.md

```diff
@@ -219,6 +226,31 @@ myNextApplication:
       apiLambda: fooApiLambda
 ```
 
+### Public asset configuration
+
+By default, assets under `/public` or `/static` don't have cache control applied. However, you can overwrite this by setting `publicAssetCache`.
```

Comment:
Can we change the default to be caching the `public/` directory for 1 year? That should be more common than not wanting to cache it. Also, users can opt out by setting it to `false`.

Reply:
yeah, sure, I implemented it with `false` as the default, since that's the current behavior, so this wouldn't be a breaking change. But sure, I can add true as the default.